ee/dtr/admin/disaster-recovery/create-a-backup.md

```diff
@@ -62,10 +62,23 @@ If you've configured DTR to store images on the local file system or NFS mount,
 you can backup the images by using ssh to log into a node where DTR is running,
 and creating a tar archive of the [dtr-registry volume](../../architecture.md):
 
+Local images:
+
+{% raw %}
+```none
+sudo tar -cf dtr-image-backup-$(date +%Y%m%d-%H_%M_%S).tar \
+  /var/lib/docker/volumes/dtr-registry-$(docker ps --filter name=dtr-rethinkdb \
```

Comment:
@caervs can you please give this an overall review and confirm the backup commands conform to best practice?